jenkins-pipelines/upgrade-scylla-operator-minikube.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
```

Comment:
rename `jenkins-pipelines/upgrade-scylla-operator-minikube.jenkinsfile` -> `jenkins-pipelines/upgrade-scylla-k8s-minikube.jenkinsfile` since from the current name one can't really understand which upgrade is testes: Scylla DB or Scylla Operator

Reply:
fixed